src/main/groovy/com/twcable/grabbit/GrabbitConfiguration.groovy

```diff
@@ -117,6 +119,26 @@ class GrabbitConfiguration {
         )
     }
 
+    /**
+     * Removes '/.' from {@code pathConfigurations}'s path if exists in the end, and Sorts the {@code pathConfigurations}
```

Comment:
Minor, but this line is documenting details that are apparent in the code. If I were a reader, I would be happy with just the line below, since it shows intent ("Sorts the pathConfigurations on the basis of path depth")